docs/services/sdk/error-handling.md

```diff
@@ -13,15 +13,39 @@ For situations where you do not want the invocation to be retried, but instead y
 to be propagated back to the caller, you can use a `terminal error`.
 
 ```typescript
-import { TerminalError } from "@restatedev/restate-sdk";
+import { TerminalError, ErrorCodes } from "@restatedev/restate-sdk";
 
 // Inside your handler throw terminal errors via:
-throw new TerminalError("Something went wrong.", { errorCode: 13, cause: "Something caused this." })
+throw new TerminalError("Something went wrong.", { errorCode: ErrorCodes.INTERNAL, cause: "Something caused this." })
 ```
 
-The `errorCode` and `cause` are optional. You can throw a terminal error anywhere in your handler.
+The `errorCode` and `cause` are optional. `errorCode` is a [gRPC status code](https://grpc.github.io/grpc/core/md_doc_statuscodes.html).
+You can throw a terminal error anywhere in your handler.
 Restate will propagate the error code and message back to the caller and will consider this invocation to be finished. 
 
+If a terminal error is returned to an HTTP client of the ingress, it will be converted into an appropriate HTTP status code
+using the following mapping:
```

Comment:
Not sure if this part of the doc with the mapping belongs here or in the TSDoc

Reply:
it is also in the tsdoc https://github.com/restatedev/sdk-typescript/pull/200

Reply:
So i would not have it here, i fear it will become quickly outdated.